### DISCUSSION TITLE
docs/develop/addon-types/userscripts/

### DISCUSSION THREAD

**Discussion by giscus** [ScratchAddons/website-v2#199] [Page comments]
# docs/develop/addon-types/userscripts/

Userscripts allow you to run code along Scratch pages - you can do stuff like add buttons, enhance the Scratch editor, or anything you can imagine.

https://scratchaddons.com/docs/develop/addon-types/userscripts/

**Comment by yoloyusof** [comment]
goofy docs

**Reply by WorldLanguages** [reply]
They definitely could be improved - if you join the development Discord server, we can usually answer any questions quickly.
